packages/auth/src/api/errors.ts

```diff
@@ -100,7 +100,8 @@ export const enum ServerError {
   MISSING_RECAPTCHA_VERSION = 'MISSING_RECAPTCHA_VERSION',
   INVALID_RECAPTCHA_VERSION = 'INVALID_RECAPTCHA_VERSION',
   INVALID_REQ_TYPE = 'INVALID_REQ_TYPE',
-  PASSWORD_DOES_NOT_MEET_REQUIREMENTS = 'PASSWORD_DOES_NOT_MEET_REQUIREMENTS'
+  PASSWORD_DOES_NOT_MEET_REQUIREMENTS = 'PASSWORD_DOES_NOT_MEET_REQUIREMENTS',
+  INVALID_HOSTING_LINK_DOMAIN = 'INVALID_HOSTING_LINK_DOMAIN'
```

Comment:
nit: if possible please add a comma at end so next time the existing line won't show up in the diff.

Reply:
I tried, but `yarn run format` automatically removed the comma.